src/geovista/core.py

```diff
@@ -706,20 +698,20 @@ def slice_cells(
 def slice_lines(
     mesh: pv.PolyData, n_points: int | None = None, copy: bool | None = False
 ) -> pv.PolyData:
-    """Cut the line mesh along the antimeridian, breaking line connectivity.
+    """Cut a line-based mesh along the Antimeridian, breaking line connectivity.
```

Comment:
The word `antimeridian` was being used to refer both to:

- The inverse of a given meridian
- The meridian at `-180`

So I wanted to make a distinction. Since the lay-person's understanding of the term would be the specific `-180` meridian, I decided capitalising that would make it clear we were referring to a specific thing rather than a concept.

Reply:
Could maybe use `180th meridian` instead, if preferred

Reply:
Cool. Refering to `Antimeridian` as a thing is totally fine here, thanks for tidying that up 😄